tests/test_app/test_composable.py

```diff
@@ -615,26 +606,6 @@ def test_user_function_multiple():
     assert got_2 == {("s1", "s2"): 2.0, ("s2", "s1"): 2.0}
 
 
-def test_appify():
```

Comment:
I am not sure that we have this test in new decorator.

Reply:
No longer a valid test for the new decorator.